manifests/integrations/eventhub-credentials-sync/_cronjobs/azure/reconcile-patch.yaml

```diff
@@ -1,27 +0,0 @@
-apiVersion: batch/v1beta1
-kind: CronJob
-metadata:
-  name: credentials-sync-eventhub
-  namespace: flux-system
-spec:
-  jobTemplate:
-    spec:
-      template:
-        spec:
-          containers:
-            - name: sync
-              image: mcr.microsoft.com/azure-cli
-              env:
-                - name: RECONCILE_SH
-                  value: |-
-                    reconcile() {
-                      echo "Starting JWT token sync -- $(date)"
-                      echo "Logging into Azure"
-                      az login --identity
-                      echo "Getting JWT token"
-                      token=$(az account get-access-token --resource https://eventhubs.azure.net |jq -r .accessToken)
-                      echo "Creating secret: ${KUBE_SECRET}"
-                      apply-secret "${KUBE_SECRET}" ${token} "${ADDRESS}"
-                      echo "Finished JWT token sync -- $(date)"
-                      echo
-                    }
```

Comment:
this entire folder being a cross-cloud piece of software, implemented in bash and kustomize, is a testament to the terrible beauty of the universe and the chaos that is allowed within it 🌌 🪐 ☄️